applications/feedback_sentiment_analyzer/SPECIFICATION.md

```diff
@@ -0,0 +1,224 @@
+# FSA Tech spec
```

Comment:
```suggestion
# Feedback Sentiment Analyzer (FSA) technical specification
```

Reply:
With PAM, we used `tech spec`. I would prefer `technical specification` but didn't want to break the established way.